src/cli-client/internals/cli/cli_main.go

```diff
@@ -0,0 +1,66 @@
+package cli
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"os"
+	"strings"
+
+	"github.com/canonical/go-flags"
+	"github.com/canonical/oci-factory/src/cli-client/internals/logger"
+)
+
+type CmdRelease struct {
+	Revision int `long:"revision" description:"The revision number of the image release"`
+}
+
+const longDescription = `The OCI Factory CLI client is a tool that builds, tests, and releases
+the OCI images owned by Canonical using the Github workflow in 
+the OCI Factory repository.
+`
+
+var opts struct {
+	// TODO: Leave `release` here for now. Should be moved into a `cmd_release.go` in phase 2.
+	// CmdRelease       CmdRelease `command:"release" description:"Release (re-tag) the image into the container registries"`
+	SkipConfirmation bool `short:"y" description:"Skip the confirmation to upload/release an image"`
+}
+
+var parser = flags.NewParser(&opts, flags.Default)
+
+func CliMain() {
```

Comment:
We should really do some error checking here. Let's return an error from this func if it does not work.